cli/installer/install-azd.ps1

```diff
@@ -61,6 +61,51 @@ param(
     [int] $DownloadTimeoutSeconds = 120
 )
 
+
+# Windows specific:
+# This functions sends a WM_SETTINGCHANGE message which causes new processes to
+# pick up the updates to environment variables. Not calling this funciton after
+# updating environment variables means that new processes will use an older view
+# of the environment variables.
+function broadcastSettingChange {
+    $SEND_MESSAGE_TIMEOUT_DEFINITION = @"
+    [DllImport("user32.dll")]public static extern IntPtr SendMessageTimeout(
```

Comment:
This is a synchronous call, which you probably don't need/want. Instead, you [`PostMessageA`](https://docs.microsoft.com/windows/win32/api/winuser/nf-winuser-postmessagea) with `WM_BROADCAST` as the first parameter.

Also, this won't likely work unless P/Invoke is doing the whole "a" vs. "w" thing, which I don't call off hand. Either redefine whatever function you use with an "A" at the end, or set the `CharSet` property. See https://docs.microsoft.com/dotnet/api/system.runtime.interopservices.dllimportattribute.charset?view=net-6.0#remarks for details.